src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,46 @@
+package duke.tasks;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event class is a class which represents the event task.
+ */
+public class Event extends Task{
+    protected LocalDate at;
```

Comment:
Similar to Deadline, perhaps here could be eventDate? This allows easier add-ons in the future if you want to add eventTime for example.